File: adminWF/Classes/Bid.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace auctionComplex.Classes
{
    [Table("bids")]
    internal class Bid
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("investment_volume")]
        public double InvestmentVolume { get; set; }

        [Column("entry_percentage")]
        public double EntryPercentage { get; set; }

        [Column("timestamp")]
        public DateTime Timestamp { get; set; }

        public User User { get; set; }
        [Column("user_id")]
        public int UserId { get; set; }

        
        public AuctionItem AuctionItem{ get; set; }
        [Column("auction_item_id")]
        public int AuctionItemId { get; set; }
    }
}

File: serverSpring/src/main/java/com/vyatsu/auctionComplex/entity/api/Bid.java
package com.vyatsu.auctionComplex.entity.api;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Entity
@Table(name = "bids")
public class Bid {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @Getter
    @Setter
    private Integer id;

    @Column(name = "investment_volume", nullable = false)
    @Getter
    @Setter
    private Double investmentVolume;

    @Column(name = "entry_percentage", nullable = false)
    @Getter
    @Setter
    private Double entryPercentage;

    @Column(name = "\"timestamp\"")
    @Getter
    @Setter
    private Instant timestamp;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "\"UserId\"")
    @JsonIgnore
    @Getter
    @Setter
    private User user;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "\"AuctionItemId\"")
    @JsonIgnore
    @Getter
    @Setter
    private AuctionItem auctionItem;

}